src/perform_step/linear_perform_step.jl

```diff
@@ -486,20 +472,17 @@ function perform_step!(integrator, cache::MagnusGL8Cache, repeat_step = false)
     mass_matrix = integrator.f.mass_matrix
     exp_method = ExpMethodGeneric()
 
-    L1 = deepcopy(integrator.f.f)
-    L2 = deepcopy(integrator.f.f)
-    L3 = deepcopy(integrator.f.f)
-    L4 = deepcopy(integrator.f.f)
+    L = integrator.f.f
     v1 = (1 / 2) * sqrt((3 + 2 * sqrt(6 / 5)) / 7)
     v2 = (1 / 2) * sqrt((3 - 2 * sqrt(6 / 5)) / 7)
-    update_coefficients!(L1, uprev, p, t - dt * v1 + dt / 2)
-    A1 = Matrix(L1)
-    update_coefficients!(L4, uprev, p, t + dt * v1 + dt / 2)
-    A4 = Matrix(L4)
-    update_coefficients!(L2, uprev, p, t - dt * v2 + dt / 2)
-    A2 = Matrix(L2)
-    update_coefficients!(L3, uprev, p, t + dt * v2 + dt / 2)
-    A3 = Matrix(L3)
+    update_coefficients!(L, uprev, p, t - dt * v1 + dt / 2)
+    A1 = deepcopy(L.A)
+    update_coefficients!(L, uprev, p, t + dt * v1 + dt / 2)
+    A4 = deepcopy(L.A)
+    update_coefficients!(L, uprev, p, t - dt * v2 + dt / 2)
+    A2 = deepcopy(L.A)
```

Comment:
This isn't correct. L.A doesn't necessarily exist. This is only specialized to a single SciMLOperator type, and instead needs to work for the general interface. Also, `Matrix(L)` on a `MatrixOperator` is a no-op to grab `L.A`, so is there a reason that wouldn't be sufficient?

Reply:
Yes, `convert(AbstractMatrix, L)` would work.